bench/DtlsBenchmark.cpp

```diff
@@ -0,0 +1,152 @@
+#include "WebRTCClientBenchmarkFixture.h"
+
+namespace com {
+namespace amazonaws {
+namespace kinesis {
+namespace video {
+namespace webrtcclient {
+
+class DtlsBenchmark : public WebRtcClientBenchmarkBase {
+  public:
+    STATUS createAndConnect(TIMER_QUEUE_HANDLE timerQueueHandle, PDtlsSession* ppClient, PDtlsSession* ppServer)
```

Comment:
This is a dup of the test code. Perhaps make WebRtcClientBenchmarkBase derive from your test instead? Or, why do you even need that base - just derive this from the test?

Reply:
I'm not sure if we want to do that because google test and google benchmark are two different frameworks, their base classes are defined differently. I'm okay with this, but I'm afraid that we'll find some unexpected caveats.